packages/search-ui/src/__tests__/actions/removeFilter.test.js

```diff
@@ -139,7 +139,46 @@ describe("#removeFilter", () => {
     ]);
   });
 
-  it("Removes all filters, if no filter value or filter type i specified", () => {
+  it("Removes a range filter value even if only the name matches", () => {
+    expect(
+      subject(
+        "test",
+        {
+          from: "1701-07-23T21:15:28.816Z",
+          name: "A really long time ago"
+        },
+        undefined,
+        {
+          initialFilters: [
+            {
+              field: "test",
+              values: [
+                {
+                  from: "1702-07-23T22:17:28.816Z",
```

Comment:
Just for my own edification - since the new logic is only checking the `name` key, what happens if the `from` key here were totally different from the `from` key on line 147 / if this line were totally removed? The test would still pass, correct?

Reply:
So, correct me if I'm wrong...

Line 147: "1701-07-23T21:15:28.816Z"
Line 157: "1702-07-23T22:17:28.816Z"

They are actually slightly different, is that what you were asking about?

I should just make them really really different so that it is more apparent.

Reply:
OH wow yeah I totally spaced and missed the `1`s that were different from the `2`s! Thats' my bad/eye fatigue or something 🤦‍♀